src/test/java/com/wikia/webdriver/testcases/portableinfoboxtests/PortableInfoboxTests.java

```diff
@@ -0,0 +1,225 @@
+    theme.selectTheme(4);
+    theme.submitThemeSelection();
+    ArticlePageObject article = new ArticlePageObject(driver);
+    article.openArticleByName(wikiURL, PageContent.PORTABLE_INFOBOX01);
+    PortableInfoboxPageObject info = article.getInfoboxPage();
+    String oldBackground = info.getBackgroundColor();
+    theme.openSpecialDesignerPage(wikiURL);
+    theme.selectTheme(1);
+    theme.submitThemeSelection();
+    article.openArticleByName(wikiURL, PageContent.PORTABLE_INFOBOX01);
+    String newBackground = info.getBackgroundColor();
+    info.verifyChangedBackground(oldBackground, newBackground);
+
+  }
+
+  @Test(groups = {"PortableInfoboxTests", "PortableInfoboxTests_012"})
+  public void verifyOrderedAndUnorderedLists() {
+    ArticlePageObject article = new ArticlePageObject(driver);
+    article.openArticleByName(wikiURL, PageContent.PORTABLE_INFOBOX02);
+    PortableInfoboxPageObject info = article.getInfoboxPage();
+    WebElement itemValue = info.getItemValue();
+    WebElement orderedListItem = info.getOrderedListItem(1);
+    info.compareFontSizes(itemValue, orderedListItem);
+    WebElement unorderedListItem = info.getUnorderedListElement(1);
+    info.compareFontSizes(itemValue, unorderedListItem);
+  }
+
+  @Test(groups = {"PortableInfoboxTests", "PortableInfoboxTests_013"})
+  public void verifyInfoboxCategoryLink() {
+    ArticlePageObject article = new ArticlePageObject(driver);
+    article.openArticleByName(wikiURL, PageContent.PORTABLE_INFOBOX01);
+    PortableInfoboxPageObject info = article.getInfoboxPage();
+    info.clickCategoryLink();
+    CategoryPageObject category = new CategoryPageObject(driver);
+    String categoryName = category.getCategoryName();
+    category.verifyCategoryPageTitle(categoryName);
+  }
+
+ @Test(groups = {"PortableInfoboxTests", "PortableInfoboxTests_014"})
+  public void verifyHorizontalGroupFontSize() {
+   ArticlePageObject article = new ArticlePageObject(driver);
+   article.openArticleByName(wikiURL, PageContent.PORTABLE_INFOBOX01);
+   PortableInfoboxPageObject info = article.getInfoboxPage();
+   WebElement horizontalItemLabel = info.getHorizontalItemLabel();
+   WebElement itemLabel = info.getItemLabel();
+   info.compareFontSizes(horizontalItemLabel, itemLabel);
+   WebElement horizontalItemValue = info.getHorizontalItemValue();
+   WebElement itemValue = info.getItemValue();
+   info.compareFontSizes(horizontalItemValue, itemValue);
+ }
+
+  @Execute (asUser = User.USER_9)
+  @Test(groups = {"PortableInfoboxTests", "PortableInfoboxTests_015"})
+  public void verifyCopiedTemplateSyntaxInArticlePresence() {
+    TemplatePageObject template = new TemplatePageObject(driver);
+    template.openArticleByName(wikiURL,
+            PageContent.PORTABLE_INFOBOX_WEBSITE_TEMPLATE);
+    ArticlePageObject article = new ArticlePageObject(driver);
+    /*TemplatePageObject template = article.openTemplatePage(wikiURL,
+            PageContent.PORTABLE_INFOBOX_WEBSITE_TEMPLATE);*/
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)